nav2_system_tests/src/system/CMakeLists.txt

```diff
@@ -56,32 +47,29 @@ ament_add_test(test_bt_navigator_2
   TIMEOUT 180
   ENV
     TEST_DIR=${CMAKE_CURRENT_SOURCE_DIR}
-    TEST_MAP=${PROJECT_SOURCE_DIR}/maps/map_circular.yaml
-    TEST_WORLD=${PROJECT_SOURCE_DIR}/worlds/turtlebot3_ros2_demo.world
-    GAZEBO_MODEL_PATH=${PROJECT_SOURCE_DIR}/models
     BT_NAVIGATOR_XML=navigate_to_pose_w_replanning_and_recovery.xml
     TESTER=nav_to_pose_tester_node.py
     ASTAR=False
     CONTROLLER=dwb_core::DWBLocalPlanner
     PLANNER=nav2_navfn_planner/NavfnPlanner
 )
 
-ament_add_test(test_dynamic_obstacle
-  GENERATE_RESULT_FOR_RETURN_CODE_ZERO
-  COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/test_system_launch.py"
-  WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
-  TIMEOUT 180
-  ENV
-    TEST_DIR=${CMAKE_CURRENT_SOURCE_DIR}
-    TEST_MAP=${PROJECT_SOURCE_DIR}/maps/map_circular.yaml
-    TEST_WORLD=${PROJECT_SOURCE_DIR}/worlds/turtlebot3_ros2_demo_obstacle.world
-    GAZEBO_MODEL_PATH=${PROJECT_SOURCE_DIR}/models
-    BT_NAVIGATOR_XML=navigate_to_pose_w_replanning_and_recovery.xml
-    TESTER=nav_to_pose_tester_node.py
-    ASTAR=False
-    CONTROLLER=dwb_core::DWBLocalPlanner
-    PLANNER=nav2_navfn_planner/NavfnPlanner
-)
+# ament_add_test(test_dynamic_obstacle
```

Comment:
Can this be readded?

Reply:
Sure, but I have to modify the world file in order to add an obstacle, the same way it was done with the old scenario. Should I add a copy of warehouse.world to `nav2_system_tests/src/system` folder?

Reply:
Sure

Reply:
Did you readd?

Reply:
Not yet. I work on it asap.

Reply:
Done. Please, have a look if it's okey for you.